CONTRIBUTING.md

```diff
@@ -73,6 +73,7 @@ below:
  - Tomasz Trzeciak (Met Office, UK)
  - Mark Worsfold (Met Office, UK)
  - Ying Zhao (Bureau of Meteorology, Australia)
+ - Nandhu S (Student, India)
```

Comment:
@iamnandhu, thank you! Would you mind moving your name higher so that the surname (S) is alphabetically sorted?